data/sidebars/concepts.yml

```diff
@@ -25,6 +25,13 @@
     - title: DDO Metadata
       link: /concepts/ddo-metadata/
 
+- group: NFTs (upcoming v4 release)
```

Comment:
Why is this made so important? We do not have a dedicated datatoken section, so why have a dedicated NFT section? It should all be part of exisitng sections, like “Specifiying assets”